CMakeLists.txt

```diff
@@ -223,6 +223,7 @@ set_target_properties(
 if(OPENSN_WITH_LUA)
     add_subdirectory(lua)
     add_subdirectory(test)
+    add_subdirectory(test/lua)
```

Comment:
This should be in `test/CMakeLists.txt` not here.

Reply:
Fixed.